src/build/strings.ts

```diff
@@ -0,0 +1,156 @@
+import { sortKeys } from '../common/objUtils';
+
+const strings = {
```

Comment:
I think some of the logic here could be simplified if you just export `strings`, then you can just reference it from any other js/ts file and resolve things automatically rather than serializing everything to stdout and rebuilding it

Reply:
We do export `strings` and use it to generate our debuggers, but we also need to create the `package.nls.json`--we do so by running strings.ts from the gulpfile. We don't want to include it as a normal module in the gulpfile, since then Node will cache it which is undesirable for the `watch` task.

Reply:
Makes sense